### PR TITLE
add-separate-disk-for-worker-nodes

### PR DESCRIPTION
**Pull request by calvix**
### Added

- Add `nonRootVolumes` to worker nodes to mount `/var/lib` and `/var/log` on separate disk volumes.

### Changed

-  control-plane nodes - Combine kubelet disk `/var/lib/kubelet` and containerd disk `/var/lib/containerd` into single disk `/var/lib` to share the volume space and safe cost.

towards https://github.com/giantswarm/roadmap/issues/3438

/run cluster-test-suites

### CODE REVIEW

File: helm/cluster-aws/Chart.yaml [hunk 1]
```diff
@@ -16,8 +16,8 @@ restrictions:
     - capa
 dependencies:
   - name: cluster
-    version: "0.19.0"
-    repository: https://giantswarm.github.io/cluster-catalog
+    version: "0.22.0-933aeb9798167ec996cc16d45cf6a599f7796d8e"
```
Comment (AndiDog) [inline helm/cluster-aws/Chart.yaml:19]: leftover

**Review by AndiDog — COMMENTED** [review]
Some comments, ~and docs must be regenerated to fix CI run~

**Review by T-Kukawka — APPROVED** [review]

**Pull request merged**